site/src/App/Navigation/Navigation.tsx

```diff
@@ -112,6 +120,30 @@ export const Navigation = () => {
       >
         <Box paddingBottom="xxlarge" marginBottom="xxlarge">
           <Outlet />
+          {config.branchName && (
+            <Box
+              position="fixed"
+              zIndex="sticky"
+              bottom={0}
+              right={0}
+              margin="medium"
+              boxShadow="small"
+              borderRadius="large"
+              background="cautionLight"
+              padding="medium"
+            >
+              <Text icon={<IconRocket />}>
+                Preview:{' '}
+                <TextLink
+                  href={`https://github.com/seek-oss/braid-design-system/compare/master...${config.branchName}`}
+                  target="_blank"
+                  weight="weak"
+                >
+                  <Strong>{config.branchName}</Strong>
+                </TextLink>
```

Comment:
Thoughts on something more descriptive than `Preview`, e.g `Preview branch:` or `Branch preview:`?

Also, comparing against `master` won't always be correct, e.g. a PR against a feature branch. Probably not worth addressing right now but worth thinking about.

Reply:
Yeah was a lazy mode to get to a diff (and that will show you the View Pull Request button if there is one). Can definitely be improved if we pass more information through in the future!

Happy to update the prefix, can chat about that for sure.